app/components/thank-you-container.js

```diff
@@ -1,9 +1,103 @@
 import Ember from 'ember';
 
 const {
-  Component
+  Component,
+  computed,
+  get,
+  isPresent
 } = Ember;
 
+const MAX_VOLUNTEERS = 12;
+
+/**
+  The `thank-you-container` component presents the main content for the
+  `thank-you` donations page. This includes an icon, thank you message and
+  a list of contributors.
+
+  ## Default Usage
+
+  ```handlebars
+  {{thank-you-container project=project}}
+  ```
+
+  @class thank-you-container
+  @module Component
+  @extends Ember.Component
+ */
 export default Component.extend({
-  classNames: ['thank-you-container']
+  classNames: ['thank-you-container'],
+
+  /**
+    A computed alias for the project's organization members
+
+    @property organizationMemberships
+    @type Ember.Array
+   */
+  organizationMemberships: computed.alias('project.organization.organizationMemberships'),
+
+  /**
+    Filters the array of `organizationMemberships` down to members who are not
+    pending acceptance.
+
+    @property nonPendingMemberships
+    @type Ember.Array
+   */
+  nonPendingMemberships: computed('organizationMemberships.[]', 'organizationMemberships.@each.role', function() {
+    let memberships = get(this, 'organizationMemberships');
+
+    if (isPresent(memberships)) {
+      return memberships.filter((membership) => {
+        let isPending = get(membership, 'isPending');
```

Comment:
agreed. one thing is that the role has to be fetched, so there is a portion of time where `role` on the memberships is `undefined`. During that loading, all memberships with `undefined` `role`s will be not `pending`.

I guess what I am getting at is the cleanest way to do it is (and the way I have changed it to):
`approvedMemberships: computed.filterBy('organizationMemberships', 'isNotPending'),`
but that does not filter properly when `role` is undefined.